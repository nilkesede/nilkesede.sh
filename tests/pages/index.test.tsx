import Index from '../../src/pages'

import {render} from '../_helpers'

it('renders correctly', () => {
  const {container} = render(<Index/>)
  expect(container).toMatchSnapshot()
})
