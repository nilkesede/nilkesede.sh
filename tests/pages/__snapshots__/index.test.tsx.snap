// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c1 {
  max-width: 1140px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 50px 30px 30px;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c3 {
  font-family: 'fontello',sans-serif;
  font-style: normal;
  font-weight: normal;
}

.c2 {
  width: 50px;
  height: 50px;
  font-size: 50px;
  padding: 0;
  margin: 0 10px;
  display: inline-block;
  line-height: 1;
  text-align: center;
}

.c0 {
  width: 100%;
  min-height: 100vh;
  margin: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div>
        <a
          class="c2"
          href="https://linkedin.com/in/nilkesede"
          rel="noopener noreferrer"
          size="50"
          target="_blank"
          title="linkedin"
        >
          <i
            class="c3"
          >
            
          </i>
        </a>
        <a
          class="c2"
          href="https://github.com/nilkesede"
          rel="noopener noreferrer"
          size="50"
          target="_blank"
          title="github"
        >
          <i
            class="c3"
          >
            
          </i>
        </a>
        <a
          class="c2"
          href="https://twitter.com/nilkesede"
          rel="noopener noreferrer"
          size="50"
          target="_blank"
          title="twitter"
        >
          <i
            class="c3"
          >
            
          </i>
        </a>
      </div>
    </div>
  </div>
</div>
`;
